conf/messages.en

```diff
@@ -202,11 +202,16 @@ homepage.p3.linkText = Create a new goods record
 homepage.h2.2 = Manage your profile
 homepage.p4.linkText = Ask for an opinion from HMRC if a commodity code is correct
 homepage.p5.linkText = Manage your profile details
-homepage.h2.3 = Help and contact
+homepage.h2.3 = Download TGP data
+homepage.h2.4 = Help and contact
 homepage.p6.linkText = Find additional support and contacts to help you
 homepage.downloadBanner.important = Important
 homepage.downloadBanner.content = The TGP records file you asked for is ready to download.
 homepage.downloadBanner.linkText = Go to the TGP records file page to download the file
+homepage.downloadLinkText.requestFile = Request a file of your TGP records
```

Comment:
AC 2.2: Messages added, Get your TGP records file ( See attached for design), You do not have TGP goods records to download. ( See attached for design) etc